sdk/core/azure-mgmt-core/azure/mgmt/core/policies/__init__.py

```diff
@@ -24,9 +24,31 @@
 #
 # --------------------------------------------------------------------------
 
+from azure.core.pipeline.policies import HttpLoggingPolicy
 from ._base import ARMAutoResourceProviderRegistrationPolicy
 
-__all__ = ["ARMAutoResourceProviderRegistrationPolicy"]
+
+class ARMHttpLoggingPolicy(HttpLoggingPolicy):
+    """HttpLoggingPolicy with ARM specific safe headers fopr loggers.
+    """
+
+    DEFAULT_HEADERS_WHITELIST = HttpLoggingPolicy.DEFAULT_HEADERS_WHITELIST | set([
```

Comment:
Let's avoid using the term WHITELIST. Call it SAFELIST or something.

Reply:
I think this may cause a breaking change as `HttpLoggingPolicy` is already using this term: https://github.com/Azure/azure-sdk-for-python/blob/d9d53128ed652a7b99d0b2cbfcab5a3683d3013e/sdk/core/azure-core/azure/core/pipeline/policies/_universal.py#L342

Reply:
I think this may cause a breaking change as `HttpLoggingPolicy` is already using this term: https://github.com/Azure/azure-sdk-for-python/blob/d9d53128ed652a7b99d0b2cbfcab5a3683d3013e/sdk/core/azure-core/azure/core/pipeline/policies/_universal.py#L342